### ISSUE TITLE
Stream failed on BBB2.4 and Greenlight 2.12.6 

### ISSUE THREAD

**Issue by netlogica** [manishkatyan/bbb-streaming#11]
Listen only button not found
TimeoutError: waiting for selector `button[aria-label="Leave audio"]` failed: timeout 30000ms exceeded
    at new WaitTask (/usr/src/app/node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:528:34)
    at DOMWorld.waitForSelectorInPage (/usr/src/app/node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:432:26)
    at internalHandler.waitFor (/usr/src/app/node_modules/puppeteer/lib/cjs/puppeteer/common/QueryHandler.js:31:77)
    at DOMWorld.waitForSelector (/usr/src/app/node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:323:29)
    at Frame.waitForSelector (/usr/src/app/node_modules/puppeteer/lib/cjs/puppeteer/common/FrameManager.js:942:51)
    at Page.waitForSelector (/usr/src/app/node_modules/puppeteer/lib/cjs/puppeteer/common/Page.js:2370:33)
    at main (/usr/src/app/stream.js:131:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
/usr/src/app/stream.js:226
        ffmpeg.kill('SIGINT');
               ^

TypeError: Cannot read properties of undefined (reading 'kill')
    at main (/usr/src/app/stream.js:226:16)

**Comment by arun-hel** [comment]
@netlogica We updated the streaming. Install new version of streaming. That should fix the issue
